### ISSUE TITLE
A11y_Visual Studio Code_Docs Setup_Link Purpose: NVDA/Narrator is not reading corresponding heading names that are present beside every '# link'. NVDA/Narrator is announcing same info for all '# links'

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-docs#4522]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_Visual Studio Code#A11ySev2#A11yTCS#A11yMAS#Chrome#DesktopWeb#Win10#Visual Studio Code Services#Link Purpose#WCAG2.4.4#NVDA#Narrator#VisualStudioCode-Web-Apr2021

### Environment details:
Application Name: Visual Studio Code
URL:[Visual Studio Code](https://code.visualstudio.com/docs/setup/linux)
Windows Version: Version 20H2 (OS Build 19042.928)
Browser: Chrome Version 90.0.4430.85 (Official Build) (64-bit)

### Note:
This issue is observed for all '# links' through out the application

### Repro Steps:

1. Turn on Narrator/NVDA
2. Hit [Visual Studio Code](https://code.visualstudio.com/docs/setup/linux) link
3. Tab till Docs link in top navigation section and press enter
4. Tab till Set up in left navigation pane and select Linux/Mac OS
5. Tab till few '# link' present beside headings (Installation/Snap etc)
6. Observe that NVDA/Narrator announces as 'number link' for all # links which is not descriptive

### Actual Result:
NVDA/Narrator is not reading corresponding heading names that are present beside every '# link'. Screen reader announces as 'number link' for all # links

### Expected Result:
NVDA/Narrator should read corresponding heading names that are present beside every # link. All # links should be descriptive enough. Screen reader should not just announce as number link

### User Impact:
Screen Reader users might not understand the purpose of link if NVDA/Narrator is not conveying proper info about the link

### Attachment:

https://user-images.githubusercontent.com/83570638/117102648-5fddc080-ad96-11eb-9a60-01b67f2f474b.mp4

https://user-images.githubusercontent.com/83570638/117102943-f7431380-ad96-11eb-9264-a73ff347e953.mp4

**Comment by kupatkar99** [comment]
Issue is fixed.
[4522_Narrator Fixed.webm](https://user-images.githubusercontent.com/98530069/218382955-e93777bb-e54b-49cb-977d-3749c954cf0d.webm)
<img width="957" alt="4522_NVDA Fixed" src="https://user-images.githubusercontent.com/98530069/218383174-7fbf4b56-ca8d-4112-961e-e2769d64af9d.png">
#Closed;
